lib/util/nist_cvr_extractor.ts

```diff
@@ -0,0 +1,41 @@
+import { ContestSelection, OptionSelection } from "../av_client/types";
+import { CVRRoot, CVRContestSelection, CVRContest } from "./nist_cvr_types";
+
+export function extractContestSelections(cvrJson): ContestSelection[] {
+  const contests = extractContestsJson(cvrJson as CVRRoot)
+
+  return contests.map(contest => {
+    return {
+      reference: contest.ContestId,
+      optionSelections: extractOptionSelections(contest.CVRContestSelection)
+    }
+  })
+}
+
+function extractOptionSelections(nistContestSelection: CVRContestSelection[]): OptionSelection[] {
+  const nistSelected = nistContestSelection.filter(nistOptionSelection => {
+    const nistSelectionPosition = nistOptionSelection.SelectionPosition[0]
```

Comment:
Do we want to ignore every other selection?

Reply:
I have implemented a guard to ensure that there is only one